projects/extension/DEVELOPMENT.md

```diff
@@ -37,13 +37,14 @@ To make changes to pgai:
    ```bash
    just ext docker-build
    ```
-1. Run the container:
+2. Run the container:
    ```bash
    just ext docker-run
    ```
-   The `projects/extension` directory is mounted to `/pgai` in the running container.
+   The root of the repo directory is mounted to `/pgai` in the running container.
+   The working directory is at `/pgai/projects/extension`.
 
-1. Work inside the container:
+4. Work inside the container:
```

Comment:
nit, we have skipped the number 3 here, and the same with the subsequent points too, all of them are +1 what they should be.

Reply:
fixed